CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `L1HandlerTrait::execute()` takes source address and payloads as arguments [MORE](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/l1_handler_execute.html)
 
+- When calling to an address which does not exists, error is forwarded to cairo runtime instead of failing the test
```

Comment:
```suggestion
- When calling to an address which does not exists, error is forwarded to Cairo instead of always failing the test. It can be now caught with `#[should_panic]`.
```